ozaria/site/components/interactive/PageInteractive/draggableStatementCompletion/index.vue

```diff
@@ -0,0 +1,166 @@
+<script>
+  import draggable from 'vuedraggable'
+
+  import StatementSlot from '../BaseDraggableSlot'
```

Comment:
For curiosity, can you explain why we need `OrderingSlots` for draggable ordering, and not for statement completion?

Reply:
^ Related to my question above in draggableOrdering

Reply:
I think the components are different groupings of the same concept. `OrderingSlots` appears to be explicitly styled for the columns.